pkgs/by-name/tp/tplay/package.nix

```diff
@@ -0,0 +1,52 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, pkg-config
+, clang
+, ffmpeg
+, openssl
+, alsa-lib
+, libclang
+, opencv
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "tplay";
+  version = "0.4.5";
+
+  src = fetchFromGitHub {
+    owner = "maxcurzi";
+    repo = pname;
```

Comment:
```suggestion
    repo = "tplay";
```

Reply:
Why should you not use rec?

Reply:
It's about not using `pname` for the repo, as it arguably isn't the same thing.